Modules/Relationships/src/Relationships.Domain/DomainErrors.cs

```diff
@@ -40,4 +40,10 @@ public static DomainError RelationshipToTargetAlreadyExists(string targetIdentit
         return new DomainError("error.platform.validation.relationshipRequest.relationshipToTargetAlreadyExists",
             $"A relationship to '{targetIdentity}' already exists. If the relationship is terminated, you can reactivate it.");
     }
+
+    public static DomainError CannotReactivateAnAlreadyRequestedReactivation()
+    {
+        return new DomainError("error.platform.validation.relationshipRequest.cannotReactivateAnAlreadyRequestedReactivation",
```

Comment:
> cannotReactivateAnAlreadyRequestedReactivation

That's bad wording. We don't want reactivate a reactivation.

Reply:
Changes made.